### PR TITLE
Livescript is Updated

### PR DESCRIPTION
**Pull request by IamRafy**
Updated

### CODE REVIEW

File: livescript.html.markdown [hunk 1]
```diff
@@ -22,7 +22,8 @@ Feedback is always welcome, so feel free to reach me over at
 [@kurisuwhyte](https://twitter.com/kurisuwhyte) :)
 
 
-```coffeescript
+coffeescript
```
Comment (vendethiel) [inline livescript.html.markdown]: ???

**Comment by vendethiel** [comment]
Seems like you're messing up this document rather than improving it.

**Comment by IamRafy** [comment]
Reverted .

**Comment by IamRafy** [comment]
Sorry . I did reverted the commit .

On Mon, Oct 19, 2015 at 7:35 PM, ven <notifications@github.com> wrote:

> Closed #1656 <https://github.com/adambard/learnxinyminutes-docs/pull/1656>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adambard/learnxinyminutes-docs/pull/1656#event-439159638>
> .
>



-- 
Best Regards ,
Mohammed Rafy
LinkedIn <https://in.linkedin.com/pub/mohammed-rafy/a2/b92/951> | Twitter
<https://twitter.com/RafyAsarmatta>  | Facebook
<https://www.facebook.com/asarmatta>
* ideabloom.wordpress.com *
+91-7204945348

**Pull request closed without merging**